lib/routes/home_route/player_route.dart

```diff
@@ -681,6 +682,7 @@ class _MainTabState extends State<_MainTab> {
         appBar: AppBar(
           elevation: 0.0,
           backgroundColor: Colors.transparent,
+          toolbarHeight: TrackPanel.height(mediaQuery.textScaleFactor) - mediaQuery.padding.top,
```

Comment:
This does change the height of the appbar, and it can be seen in generated goldens as well

Reply:
True, this is setting the height so it doesn't get too small if the system status bar is not visible. I thought the idea was that the app bar would be the exact same height as the `TrackPanel`, since one fades over the other when opening the player route.
It shouldn't be a problem if the app bar is larger, but it must not be smaller. What do you think about this?
```suggestion
          toolbarHeight: max(
              TrackPanel.height(mediaQuery.textScaleFactor) - mediaQuery.padding.top,
              theme.appBarTheme.toolbarHeight ?? kToolbarHeight
          ),
```